docs/rsdl/rapid-rsdl-paths.md

```diff
@@ -0,0 +1,103 @@
+---
+id: rsdl-paths
+title: RAPID Paths
+---
+
+# Paths in a RAPID Service
+
+The set of valid paths for a RAPID service is the result of applying common path conventions to the model, as described below. You can optionally specify an subset of these paths that are supported by your service using [explicit paths](rapid-rsdl-intro.md#explicit-paths) in your RSDL.
+
+- [Service Root](#service-root)
+- [Key Segment](#key-segment)
+- [Structured Properties](#structured-properties)
+- [Nested Operations](#nested-operations)
+
+## Service Root
+
+The [service](./rapid-rsdl-intro.md#defining-a-service) element in RSDL defines the segments that are available at the service root:
+
+```rsdl
+service {
+  company: Company
+}
+```
+
+## | supported path |
+
+`/company`
+
+## Key Segment
+
+Path segments that represent a collection of [structured types](rapid-rsdl-intro.md#defining-a-structured-type) with identity may be followed by a segment containing the key value of an instance within the collection:
```

Comment:
The text "containing the key value of an instance" would imply that one has to put a concrete potential key value in the path, for example `/companies/MSFT`. 

Instead the path has to contain a placeholder segment.